frontend/public/components/marketplace/marketplace-item-modal.jsx

```diff
@@ -31,7 +31,7 @@ const MarketplaceItemModal = (props) => {
       <Modal.Body>
         <div className="co-catalog-page__overlay-body">
           <PropertiesSidePanel>
-            <Button bsStyle="primary" className="co-catalog-page__overlay-create" /* onClick={ openSubscribe }*/ >
+            <Button bsStyle="primary" className="co-catalog-page__overlay-create" onClick={() => openSubscribe()} >
```

Comment:
Make this link to `/k8s/ns/${props.operatorGroup.data[0].metadata.namespace}/subscriptions/new?pkg=${item.name}&catalog=${item.obj.status.catalogSource}&catalogNamespace=${item.obj.status.catalogSourceNamespace}`.